From 3fead584fa2939775d71e5aa70cd4eed013db67b Mon Sep 17 00:00:00 2001
From: Piotr Gorski <lucjan.lucjanov@gmail.com>
Date: Wed, 8 Dec 2021 13:53:05 +0100
Subject: [PATCH 5/7] Revert "drm/i915: Add workaround numbers to
 GEN7_COMMON_SLICE_CHICKEN1 whitelisting"

This reverts commit 8e6455bb06f2f5ec7acf09f964d3adc823098e08.

Signed-off-by: Piotr Gorski <lucjan.lucjanov@gmail.com>
---
 drivers/gpu/drm/i915/gt/intel_workarounds.c | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/drivers/gpu/drm/i915/gt/intel_workarounds.c b/drivers/gpu/drm/i915/gt/intel_workarounds.c
index 60e8896..6b5ab19 100644
--- a/drivers/gpu/drm/i915/gt/intel_workarounds.c
+++ b/drivers/gpu/drm/i915/gt/intel_workarounds.c
@@ -1513,11 +1513,7 @@ static void tgl_whitelist_build(struct intel_engine_cs *engine)
 				  RING_FORCE_TO_NONPRIV_ACCESS_RD |
 				  RING_FORCE_TO_NONPRIV_RANGE_4);
 
-		/*
-		 * Wa_1808121037:tgl
-		 * Wa_14012131227:dg1
-		 * Wa_1508744258:tgl,rkl,dg1,adl-s,adl-p
-		 */
+		/* Wa_1808121037:tgl */
 		whitelist_reg(w, GEN7_COMMON_SLICE_CHICKEN1);
 
 		/* Wa_1806527549:tgl */
-- 
2.34.1.75.gabe6bb3905

